internal/rpchelp/helpdescs_en_US.go

```diff
@@ -97,6 +97,10 @@ var helpDescsEnUS = map[string]string{
 	"createsignature-hashtype":              "The signature hash flags to use.",
 	"createsignature-previouspkscript":      "The the hex encoded previous output script or P2SH redeem script.",
 
+	// DisapprovePercentCmd help.
+	"disapprovepercent--synopsis": "Returns the wallet's current block disapprove percent. i.e. 100 means that all blocks are disapproved. Only used for testing purposes.",
+	"disapprovepercent--result0":  "The disapprove percent. When voting, this percent of blocks with be randomly disapproved.",
```

Comment:
As is, this isn't exactly right: the disapproval percentage is per ticket (as in: x% chance of voting to disapprove with a winning ticket) vs chance of completely disapproving a block.

I guess this is more flexible than the original idea, since it also allows testing scenarios where some tickets vote to approve and some vote to disapprove (while setting to 100% still guarantees a block disapproval).

Reply:
You can't really guarantee the block disaproval rate, unless you are the only voter. Will adjust the help here to be more explicit.

Reply:
This any better?

Reply:
LGTM